frontend/src/exporter/types.ts

```diff
@@ -4,6 +4,7 @@ export enum ExportType {
     Image = 'image',
     Embed = 'embed',
     Scene = 'scene',
+    Login = 'login',
```

Comment:
I'm being nitpicky but over time this will get confusing - I really would prefer "unlock" for this kind of thing. Login should be reserved for accounts / sessions